tests/gha-run-k8s-common.sh

```diff
@@ -128,7 +130,8 @@ function create_cluster() {
 		--node-count 1 \
 		--generate-ssh-keys \
 		--tags "${tags[@]}" \
-		$([[ "${KATA_HOST_OS}" = "cbl-mariner" ]] && echo "--os-sku AzureLinux --workload-runtime KataMshvVmIsolation")
+		$([[ "${KATA_HOST_OS}" = "cbl-mariner" ]] && echo "--os-sku AzureLinux --workload-runtime KataMshvVmIsolation") \
+		$([ -n "${AZ_NODEPOOL_TAGS}" ] && echo "--nodepool-tags "${AZ_NODEPOOL_TAGS}"")
```

Comment:
may introduce a new shellcheck warning?

Reply:
```
./tests/gha-run-k8s-common.sh:134:5: note: Prefer [[ ]] over [ ] for tests in Bash/Ksh/Busybox. [SC2292]
./tests/gha-run-k8s-common.sh:134:60: warning: The surrounding quotes actually unquote this. Remove or escape them. [SC2027]
./tests/gha-run-k8s-common.sh:134:60: note: Double quote to prevent globbing and word splitting. [SC2086]
```
if I am not seeing this wrong